How to update translations

I’d like to update the Polish translation, but I can’t make use of the Makefile in po/:

```
$ make
find: ‘../data’: No such file or directory
find: ‘../data’: No such file or directory
rm -f ./alexandria.pot
Updating PO template from Ruby files
make: rgettext: Command not found
make: *** [Makefile:15: alexandria.pot] Error 127
```

Is there a way to get a fresh .pot file for Alexandria?

--- a/po/Makefile
+++ b/po/Makefile
@@ -3,7 +3,7 @@ POT=./alexandria.pot
 
 RUBY=../bin/alexandria $(shell find ../lib -name '*.rb' -and -not -path '*macui*')
 
-GLADE=$(shell find ../data -name '*.glade')
+GLADE=$(shell find ../share/alexandria/glade -name '*.glade')
 
 HEADERS=$(shell find .. -name '*.h')
 
@@ -12,7 +12,7 @@ PO=$(wildcard *.po)
 $(POT): $(RUBY) $(GLADE) ../alexandria.desktop.in.h $(HEADERS)
 	rm -f $(POT)
 	@echo Updating PO template from Ruby files
-	@rgettext --output=$(POT) $(RUBY)
+	@rxgettext --output=$(POT) $(RUBY)
 	@echo Updating PO template from Glade files
 	@xgettext --output=$(POT) --join-existing $(GLADE)
 	@echo Updating PO template from header files

